paymentsheet/src/main/java/com/stripe/android/paymentsheet/NotNullLiveData.kt

```diff
@@ -0,0 +1,10 @@
+package com.stripe.android.paymentsheet
+
+import androidx.lifecycle.MutableLiveData
+
+internal class NotNullMutableLiveData<T>(value: T) : MutableLiveData<T>(value) {
+    @Suppress("UNCHECKED_CAST")
+    override fun getValue(): T = super.getValue() as T
+    override fun setValue(value: T) = super.setValue(value)
+    override fun postValue(value: T) = super.postValue(value)
+}
```

Comment:
Why is this necessary?

Reply:
This allows us to expect a MutableLiveData is never a null value

Reply:
See comment below.